main.rb

```diff
@@ -19,19 +31,20 @@ def list_of_options
 end
 
 def option(input)
+  app = App.new
   case input
   when 1
-    list_all_persons
+    app.list_all_persons
   when 2
-    list_all_books
+    app.list_all_books
   when 3
-    list_all_rentals
+    app.list_all_rentals
   when 4
-    add_person
+    app.add_person
   when 5
-    add_book
+    app.add_book
   when 6
-    add_rentals
+    app.add_rentals
   else
     puts 'Invalid option. Please try again.'
```

Comment:
- [ ]  Great job to optimize your app, but you still have some issues you need to fix, Seemed like you've not match your features from app with the one from the referencing videos, for example, in the video  3 => create a person, but for you   3=> list rentals. please try to match the method with corresponding numbers as it is provided videos.